File: Niel/first_year/progamming/V25/in1010/Oppgaver/trix/Handletur.java
import java.util.Scanner;


class Handletur {
    public static void main (String[] args){
        

        int Broed = 20;
        int Melk = 15;
        int Ost = 40;
        int Youghurt = 12;

        int total = 0;

        System.out.println("");
        System.out.println("Hei! Velkommen til IFI-butikken.");

        System.out.println("Hvor mange broed vil du ha?");
        Scanner bruker_broed = new Scanner(System.in);
        int antall_broed = bruker_broed.nextInt();
        total += antall_broed*Broed;


        System.out.println("Hvor mange Melk vil du ha?");
        Scanner bruker_Melk = new Scanner(System.in);
        int antall_Melk = bruker_Melk.nextInt();
        total += antall_Melk*Melk;

        System.out.println("Hvor mange Ost vil du ha?");
        Scanner bruker_Ost = new Scanner(System.in);
        int antall_Ost = bruker_Ost.nextInt();
        total += antall_Ost*Ost;

        System.out.println("Hvor mange Youghurt vil du ha?");
        Scanner bruker_Youghurt = new Scanner(System.in);
        int antall_Youghurt = bruker_Youghurt.nextInt();

        total += antall_Youghurt*Youghurt;

        System.out.println("Totall pris: "+ total +"kr.");

    }
}

File: Niel/first_year/UiO/H24/oving/02/handletur.py
# - 02.18: Handletur

melk = 15
broed = 20
ost = 40
yoghurt = 12

sum = 0

print("\nHei! Velkommen til IFI-butikken.\n")
abroed = int(input("Hvor mange brød vil du ha? \n> "))
amelk = int(input("Hvor mange melk vil du ha? \n> "))
aost = int(input("Hvor mange ost vil du ha?\n >"))
ayog = int(input("Hvor mange yoghurt vil du ha?\n >"))

 
sum = melk*amelk + ost*aost + yoghurt*ayog + broed*abroed

print(f"Du skal betale: {sum} kr.\n")